packages/pytest-simcore/src/pytest_simcore/helpers/faker_factories.py

```diff
@@ -260,6 +261,7 @@ def random_product(
             ui=VendorUI(
                 logo_url=fake.url(),
                 strong_color=fake.color(),
+                project_alias=random.choice(["project", "study"]),
```

Comment:
For consistency with the rest of the code, rather use `fake` model to produce a random choice
```python
project_alias = fake.random_element(elements=["project", "study"])
```